src/webprojects/fxa.md

```diff
@@ -0,0 +1,115 @@
+* Make changes to some of the settings, check for possible error messages or warnings; check for followup confirmation emails.
+
+### Testing the subscription platform
+
+The subscription platform consists of a few features. Right now only the _Payment and subscription creation_ is available for testing:
+
+* _Payment and subscription creation_: The initial payment funnel where users enter their credit card information to subscribe to a product. Mozilla partners with Stripe to handle this.
+* _Billing_: Ongoing billing and receipt emails once a subscription is entered.
+* _Subscription management UI_: Settings that allow the user to update their payment information.
+
+Payment testing is tied to the language of user preference and location. Payment service is market specific. For instance, the product is available in German, but the service is available in Germany and not in Switzerland.
+
+#### Before testing
+
+* Create a new Firefox account for testing purpose. Your existing account may or may not work.
+* Choose a Firefox browser for desktop, Android, or iOS.
+* Set the language you are testing in. Find out how to configure to the correct display language by following the steps detailed in this [SUMO article](https://support.mozilla.org/kb/choose-display-languages-multilingual-web-pages).
+* Have the link ready for the [subscription testing server](https://accounts.stage.mozaws.net/subscriptions/products/prod_HEJ13uxjG4Rj6L?plan=plan_HEJ1l12p1taV7I). Note, this link is for testing subscription only, different from the FxA staging server.
+
+**Tips:** Feeling not having enough test accounts to use for testing? You are not alone. Here are some tips you can consider.
+
+* Add number(s) or letter(s) to your existing account to create more accounts by adding a `+` in your email address. For example, if your main email is `name@xyz.com`, your expanded accounts could be `name+1@xyz.com` and `name+m@xyz.com`. You will receive email notifications in your `name@xyz.com` inbox. You can add anything you want, several digits and/or letters. Note that this works with Gmail and Google Apps domains.
+* Keep your password simple so you can log in and log out easily. Keep track of which account is connected with which password. Of course, there is Lockwise for the rescue!
+
+#### Testing payment and subscription creation
+
+* Go to the [subscription testing server](https://accounts.stage.mozaws.net/subscriptions/products/prod_HEJ13uxjG4Rj6L?plan=plan_HEJ1l12p1taV7I)
+* Click on the **Subscribe** button which takes you to a subscription form. If you don't have a Firefox account at this point or if your existing account doesn't work, you will need to create one.
+* Fill out all the required fields in the form.
+* Key in credit card number `4242 4242 4242 4242`. The mandatory fields for Expiration date, CCV and postal code can be of any random information.
+* Check formatting accuracy for date, currency and postal code. Check the box on usage agreement.
```

Comment:
Check formatting -> Verify formatting

The second check becomes confusing otherwise.